R/utils-render.R

```diff
@@ -110,6 +110,10 @@ startTemplatedRender <- function(reportJson, author){
   #call custom render function for the report, it should turn a list of HTML fragments that will available to the templates
   templateData[["renderedFragments"]] <- renderCustomFragments(reportJson)
   
+  #adding appropriately formatted start and end dates to reportJSON
+  templateData[['reportMetadata']][['displayStartDate']] <- format(flexibleTimeParse(templateData[['reportMetadata']][['startDate']], templateData[['reportMetadata']][['timezone']]), "%Y-%m-%d")
```

Comment:
instead of changing startDate and endDate here, because I'm not sure if other downstream functions require those to include the times, I created a new displayStartDate and displayEndDate